### PR TITLE
Cross compile Potassium FRC to Scala Native

### PR DESCRIPTION
**Pull request by shadaj**
The Scala Native 🚢  is ready to sail!

### CODE REVIEW

File: build.sbt [hunk 1]
```diff
@@ -97,12 +97,26 @@ lazy val vision = project.dependsOn(coreJVM).settings(
...
+  libraryDependencies ++= jvmDependencies,
+  resolvers += "WPILib-Maven" at "http://team846.github.io/wpilib-maven",
+
+  libraryDependencies += "edu.wpi.first" % "wpilib" % "2018.1.1",
+  libraryDependencies += "edu.wpi.first" % "ntcore" % "2018.1.1",
+  libraryDependencies += "com.ctre" % "phoenix" % "5.1.3.1"
+).nativeSettings(
+  resolvers += "Funky-Repo" at "http://team846.github.io/repo",
+  libraryDependencies += "com.lynbrookrobotics" %%% "wpilib-scala-native" % "0.1-SNAPSHOT",
+//  libraryDependencies += "com.lynbrookrobotics" % "ntcore" % "2018.1.1"
```
Comment (PhilipAxelrod) [inline build.sbt:113]: Is this a mistake?

**Review by PhilipAxelrod — CHANGES_REQUESTED** [review]

File: build.sbt (hunk 1 above)
Comment (shadaj) [reply build.sbt:113]: Just a reminder that at some point I actually need to make networktables run on Scala Native.

Comment (PhilipAxelrod) [reply build.sbt:113]: Oh, I see, this is in `nativeSettings`

**Review by PhilipAxelrod — APPROVED** [review]
lgtm

**Pull request merged**